bpf/bpf_host.c

```diff
@@ -1069,15 +1067,8 @@ do_netdev(struct __ctx_buff *ctx, __u16 proto, __u32 __maybe_unused identity,
 # endif /* defined(ENABLE_HOST_FIREWALL) */
 
 # ifdef ENABLE_WIREGUARD
-		if (!from_host) {
-			next_proto = ip6->nexthdr;
-			hdrlen = ipv6_hdrlen(ctx, &next_proto);
-			if (likely(hdrlen > 0) &&
-			    ctx_is_wireguard(ctx, ETH_HLEN + hdrlen, next_proto, identity)) {
-				trace.reason = TRACE_REASON_ENCRYPTED;
```

Comment:
It looks like we're no longer setting `trace.reason` to `TRACE_REASON_ENCRYPTED`. Is that intentional?